### PR TITLE
Fixes #10, broken build after checkout because of missing certificate configuration

### LINKED ISSUE #10: Can't build: Execution failed for task ':signArchives'
**Summary:** Can't build locally after checking out the project

**Steps to reproduce:**

- Checkout `scenarioo-java`
- Execute `./gradlew clean build` 

**Expected result:** Successful build

**Actual result:**
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':signArchives'.
> Cannot perform signing task ':signArchives' because it has no configured signatory
```

### PR DESCRIPTION
**Pull request by cgrossde**
Taken from: https://github.com/stianh/gradle-jaxb-plugin/commit/ff912752fe7d83013497305fe038652b530b6866

This enables other developers to locally do a gradle install of their own versions without being stopped because the build want to sign the artifacts first.

To enable signing now, one must set the project property sign, like this:

gradle -Psign

### CODE REVIEW

**Comment by bruderol** [comment]
@cgrossde Thanks. Great! 

@danielsuter can you please review and give feedback if you see any problems with that change after you return - but we will merge it now, once the docu is adjusted.

**Review by bruderol — CHANGES_REQUESTED** [review]
please adjust docu in docs folder for building a release with signed flag. then it can be merged in my oppinion,

**Review by adiherzog — APPROVED** [review]
Reviewed code changes with @cgrossde. Tested the change successfully locally. Extended the documentation.

**Pull request merged**
